### PR TITLE
Corrected the docs per the changes from the spanmetrics processor to spanmetrics connector

### PR DESCRIPTION
**Pull request by jkowall**
Updating the docs per changes in Otel. I can also add a link to @pavolloffay 's tutorial, might be a good set of examples for users in the doc. Just let me know.

### CODE REVIEW

**Review by yurishkuro — CHANGES_REQUESTED** [review]
It looks like all line endings are changed, while the text changes are minor. Please make sure to keep the original line endings.

File: content/docs/next-release/spm.md
Comment (albertteoh) [inline content/docs/next-release/spm.md:1]: There are some breaking changes introduced in the spanmetrics connector. As such, I suggest we address https://github.com/jaegertracing/jaeger/issues/4345 first before we merge this change into documentation.

**Review by albertteoh — COMMENTED** [review]
Thanks for putting this together, Jonah. I've added a comment about the timing of this change.

**Comment by jkowall** [comment]
> It looks like all line endings are changed, while the text changes are minor. Please make sure to keep the original line endings.

I am not sure what you mean by line endings? I am also not sure why the full file changed when I only edited a few things. Probably has to do with copying files around.

**Comment by yurishkuro** [comment]
I mean Unix v windows new-line

**Comment by yurishkuro** [comment]
@jkowall I ran `dos2unix` on the file and it fixed the line endings.

@albertteoh is this PR still needed, or does the one you made already covers this?

**Comment by albertteoh** [comment]
> @albertteoh is this PR still needed, or does the one you made already covers this?

The PR I made should cover this PR.

**Pull request closed without merging**